partials/footer.hbs

```diff
@@ -0,0 +1,76 @@
+<footer class="grid px-8 md:grid-cols-6 md:px-16 py-12 bg-black text-white text-sm">
```

Comment:
Can't remember if we explicitly discussed this, but let's use flexboxes instead.

On my screen the footer ends up aligned towards the left, would be better to do a center align if possible.

![image](https://user-images.githubusercontent.com/2070423/121498592-23455a80-ca0f-11eb-8c3a-308a2242dc8a.png)

Reply:
yup, I have swapped out the grid system for flex in the new commit